code/modules/mob/inventory.dm

```diff
@@ -332,3 +332,5 @@ var/list/slot_equipment_priority = list( \
 	for(var/entry in get_equipped_items(include_carried))
 		drop_from_inventory(entry)
 		qdel(entry)
+

```

Comment:
butwhy

Reply:
I was going to go down the rabbit hole of porting TG's procs for stuff, then realized that there was a better way of doing it, so I got rid of the edits, except this line apparently. ree.